src/main/java/baseball/BaseballGameController.java

```diff
@@ -0,0 +1,36 @@
+package baseball;
+
+public class BaseballGameController {
+
+    public BaseballGameModel guess(Player computer, String input) {
+        NumberBoard computerAnswer = computer.getNumberBoard();
+        NumberBoard playerAnswer = new NumberBoard(input);
+
+        int[] ballCount = countBallByAnswer(computerAnswer, playerAnswer);
+        int size = computerAnswer.size();
+        return new BaseballGameModel(ballCount, size);
+    }
+
+    private int[] countBallByAnswer(NumberBoard computerAnswer, NumberBoard myAnswer) {
+        int strike = 0;
+        int ball = 0;
+        int size = computerAnswer.size();
+
+        for (int i = 0; i < size; i++) {
+            for (int j = 0; j < size; j++) {
+                if (computerAnswer.get(i) == myAnswer.get(j) && i == j) {
+                    strike++;
+                    break;
+                }
+                if (computerAnswer.get(i) == myAnswer.get(j) && i != j) {
+                    ball++;
+                    break;
+                }
+            }
+        }
+        int[] result = new int[2];
+        result[0] = strike;
+        result[1] = ball;
```

Comment:
```suggestion
        int[] result = { strike, ball };
```
배열에 대해서는 저도 부정적인 입장이긴 합니다.
그래도 일단 요 코드에 대해서 이렇게 조금 깔끔하게 써볼 수 있을 것 같아요.

Reply:
맞아요 더 직관적이고 깔끔하네요! 감사합니다